src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -7,9 +7,8 @@
     <div class="panel panel-default">
       <div class="panel-heading">
         <h3 class="panel-title">
-          <span
-            i18n>{mode, select, editing {Edit} cloning {Clone} copying {Copy} other {Add}}</span>
-          RBD
+          <span i18n>{mode, select, editing {{{ActionLabels.EDIT}}} cloning {{{ActionLabels.CLONE}}} copying {{{ActionLabels.COPY}}} other {{{ActionLabels.CREATE}}}}</span>
+          Image
```

Comment:
Thanks, @tspmelo! Yup, that's exactly what I was doing right now (@alfonsomthd warned me about this):
```html
<span i18n="@@formTitle">{{action | titlecase}} {{resource | titlecase}}</span>
```

And feeding translated `action` and `resource`.

Reply:
That might look strange in transifex, a translation with just 2 untranslatable blocks, but it will be useful for languages that change the order of the sentence.

Reply:
I was wondering exactly the same: by leaving i18n on the span block, the whole string can still be accessible for a translator. For making this clear, I'll add a i18n description to aid on the translation.